samples/bluetooth/mesh/light/boards/nrf54l15pdk_nrf54l15_cpuapp.conf

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+################################################################################
+# Application overlay - nrf54l15
+
+CONFIG_PM_SINGLE_IMAGE=n
```

Comment:
The sample works fine (probably after PM fix) without this option. Why should we touch it now?

Reply:
Hm... It always shows a settings initialization error. Did you run it on my branch?

Reply:
I pulled main branch and build it. No warning except the partial erase option.

Reply:
It is built normally - yes. However, when you run device it cannot pass settings initialization.

Reply:
It worked as well on my desk. Please try again.